opentracing/defaulttracer.go

```diff
@@ -0,0 +1,57 @@
+package opentracing
+
+var (
+	defaultOpenTracer OpenTracer = noopOpenTracer{noopTraceContextSource{}}
```

Comment:
Given the convenience the methods in this file provide, I think we can assume most frameworks will be using them, tus relying on a global tracer instance of Construction API. For tracing systems willing to plug-in at the construction API rather than at the Raw API (e.g. due to performance considerations), I think we should have a way to set the default tracer explicitly, not just via `NewStandardTracer(recorder, ctxSource)`